README.md

```diff
@@ -17,7 +17,7 @@ With Kedro-Viz you can:
 - Get a clear picture when you have lots of datasets and nodes by using tags to visualise sub-pipelines
 - Search for nodes and datasets
 
-![Kedro-Viz Pipeline Visualisation](https://github.com/quantumblacklabs/kedro-viz/blob/master/.github/img/pipeline_visualisation.png?raw=true)
+![Kedro-Viz Pipeline Visualisation](https://github.com/quantumblacklabs/kedro-viz/blob/develop/.github/img/pipeline_visualisation.png?raw=true)
```

Comment:
Why change it to point at the develop branch?

Reply:
We ran into this issue on Kedro, when it points to `develop` you can immediately see the change without having to wait for a release because your default branch is set to `develop`.

Reply:
Gotcha, fair enough! Feels weird to me because I want it to be showing the master readme, but seeing as it has to show the `develop` readme, it makes sense.